src/Car/CarSearchService.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Car;
+use PDO;
+use Exception;
+
+class CarSearchService {
+	private CarRepository $carRepository;
+	private CarView $carView;
+	private \Location\LocationRepository $locationRepository;
+
+	public function __construct(PDO $connection) {
+		$this->carRepository = new CarRepository($connection);
+		$this->carView = new CarView();
+		$this->locationRepository = new \Location\LocationRepository($connection, $this->carRepository);
+	}
+
+	public function searchCars($string) {
+        header("Content-Type: application/json");
+        $recherche = explode(" ", $string);
+        $marque = $recherche[0];
+        if(sizeof($recherche) >= 2)
```

Comment:
sizeof est un alias de count, et ça évite des confusions avec la méthode homonyme en C.

```suggestion
        if(count($recherche) >= 2)
```